### DISCUSSION TITLE
Why won't the Shared/edited dates tooltip addon work for me?

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#4640] [Help] [answered]
I couldn't see any sort of tooltip with additional information for my or others' projects when using the `last-edit-tooltip` (Shared/edited dates tooltip) addon. I was hovering over the share date below the project on both My Stuff and the project page, but nothing happened. Am I not using this right or is there a bug?

Windows 10 with Edge 101.0.4951.64 and Scratch Addons 1.26.0

**Comment by TheColaber** [comment, marked as answer]
Can you open the console when the addon doesn't work (Ctrl shift I) and see if there are any red errors? Maybe also try reloading the page? That addon can't be "dynamicly" enabled.

**Reply by DNin01** [reply]
I don't see anything that catches my eye. But I did hit refresh and... hey, it just started working. (How did that happen?) But it still seems inconsistent sometimes. I expected it to at least start working after I click a project without requiring a refresh.
